bridges/snowbridge/primitives/inbound-queue/src/v2/message.rs

```diff
@@ -169,6 +194,26 @@ impl TryFrom<&Log> for Message {
 			}
 		}
 
+		let xcm = match payload.xcm.kind {
+			0 => XcmPayload::Raw(payload.xcm.data.to_vec()),
+			1 => {
+				let create_asset = IGatewayV2::XcmCreateAsset::abi_decode(&payload.xcm.data, true)
+					.map_err(|decode_err| {
+						log::debug!(
```

Comment:
ditto, I wonder about the utility of these debugging statements.

Reply:
Addressed in https://github.com/vgeddes/polkadot-sdk/pull/13/commits/242d9ee4310ac6c494a9329e66c66de4460197ee.